.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Test
+on:
+  push:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Node v${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+    strategy:
+      matrix:
+        node-version: [16, 18]
```

Comment:
I think we don't need to test on node `16` as it's not the LTS anymore. 18 should be okay for now.

Reply:
I will change it to 18. Keeping this matrix, just in case we need to add more versions.